### DISCUSSION TITLE
颱風假

### DISCUSSION THREAD

**Discussion by Jing-Asteroid** [1101-GS4719-C/Python-Community-GS4719-C#2137] [General]
基於近期的颱風假事件，明明氣象預報感覺會全台受災，但是因為諸多因數使得北部大部分地區的颱風假是在電影院、KTV避難，而南部居民卻在水深火熱當中度過。
現在颱風假放與不放也很常成為政治口水戰，不放假有人罵但放假也會有人噴。如果是各位變成各地首長的話會有什麼樣的想法促使你決定要放假還是不放假呢?想聽聽大家的看法。

**Comment by 112121py** [comment]
首先最重要的當然是事先訂的風量雨量標準，再來就是要以過往經驗評估這個地區的天災危害程度，有的地方可能會比較容易在風雨不大就形成嚴重災害，最後就是針對颱風的預測，如果這個颱風非常詭譎，那當然為了民眾安全還是會選擇放假，但像這次的颱風，好像也很少人想到/預測到他會停在那的轉啊轉不進來，導致最後北部的災情沒有想像中那麼嚴重，這其實也蠻沒有辦法的(?
但當然，放假這種事情一定沒辦法做到讓所有人都滿意，不管今天放不放都會有人罵，所以會調整好自己的心態，以民眾安全為主還有為天災做準備及復原來考量。
